docs/openapi.yml

```diff
@@ -404,6 +404,37 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  /cohorts/{id}:
```

Comment:
this `GET /cohorts/{id}` tells me that the endpoint should be GET Cohort data by ID (ie. all cohort data: users, cohort name, start date, etc...), not just the users.

if you want to just do GET cohort users, then the endpoint route should be: `GET /cohorts/{id}/users`

Reply:
There is no other data in cohorts other than users, but I can do this for extensibility